packages/manager/apps/pci-private-network/src/api/utils/utils.ts

```diff
@@ -37,3 +37,12 @@ export const getAutoGeneratedName = (prefix = ''): string => {
 
   return `${prefixPart}${date}-${randomNumber}`;
 };
+
+export function isValidCidrMask(mask: number): boolean {
+  return mask >= 1 && mask <= 32;
+}
+
+export function isValidIpAddress(ip: string): boolean {
+  const regex = /^((25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)\.){3}(25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)$/;
+  return regex.test(ip);
+}
```

Comment:
**question:** Instead of re-inventing the wheel, can we use the lib https://www.npmjs.com/package/ipaddr.js for IP validations?

Reply:
@anooparveti I agree with @lionel95200x I think it is not necessary to import an entire library for a single function. For now, I think he'll do the job but I'll add a unit test according @SimonChaumet 's remark